test/components/articles/available_article.spec.jsx

```diff
@@ -7,7 +7,7 @@ describe('AvailableArticle', () => {
   const props = {
     course: { home_wiki: { language: 'en', project: 'wikipedia' } },
     assignment: { article_title: 'two' },
-    current_user: { role: 0 }, // student role
+    current_user: { isStudent: true }, // student role
```

Comment:
How do I do such a thing for `course_page_spec.rb:279`?

Reply:
You create two CoursesUsers records, one for each role.

Reply:
There shouldn't be any changes to the ruby specs needed, that I can think of.

Reply:
`course_page_spec.rb` was failing somehow.

Reply:
Is my code alright in the first place? With the replacing the whole thing with just two mutually exclusive if conditions?

Reply:
Not sure. I'll check it out today.

Reply:
I think, I should just use role itself. 

if(role == 0)
  showSelect
if(role == 1 || course.user.admin)
  showRemove